subgraphs/apeswap/src/utils/pricing.ts

```diff
@@ -0,0 +1,211 @@
+import { Address, BigDecimal, BigInt, dataSource, log } from "@graphprotocol/graph-ts";
+import { Bundle, LiquidityPool, Token } from "../../generated/schema";
+import { BIGDECIMAL_ONE, BIGDECIMAL_ZERO, BSC, factoryContract, POLYGON, ZERO_ADDRESS } from "./constant";
+import { getOrCreateToken } from "./token";
+
+const WBNB_ADDRESS = dataSource.network() == "bsc" ? BSC.WBNB_ADDRESS : POLYGON.WMATIC_ADDRESS;
```

Comment:
you can't just call everything WBNB. maybe use a name like `WRAPPED_NATIVE_TOKEN_ADDRESS` and have a comment explaining

Reply:
Same for other places where `BNB` is used